quarkus/defaults/src/main/resources/application.properties

```diff
@@ -130,6 +131,8 @@ polaris.authentication.token-broker.max-token-generation=PT1H
 %test.quarkus.log.category."org.apache.polaris.service.storage.PolarisStorageIntegrationProviderImpl".level=ERROR
 %test.quarkus.http.limits.max-body-size=1000000
 %test.quarkus.otel.sdk.disabled=true
+%test.polaris.authentication.authenticator.type=test
```

Comment:
My idea was to do exactly the opposite: make all tests (automated + regression) use the default authenticator. Cf #804.

Reply:
That makes sense to me.